features/GEM052_Correct-geometric-subcontexts.feature

```diff
@@ -0,0 +1,22 @@
+@implementer-agreement
+@GEM
+@version1
+@E00050
+Feature: GEM052 - Correct geometric subcontexts
+The rule verifies that there is a minimum of at least one subcontext per context. 
+Reference: https://github.com/buildingSMART/Sample-Test-Files/issues/137.
+The context
+identifier should be one of the agreed values:
+
+Scenario: Each geometric context must have a subcontext
+    
+    Given an IfcGeometricRepresentationContext without subtypes
+    
+    Then HasSubContexts = not empty
+
+
+Scenario: Constraints on context type
+    Given An IfcGeometricRepresentationSubContext
+
+    Then ContextIdentifier = 'Annotation' or 'Axis' or 'Box' or 'Footprint' or 'Reference' or 'Body' or 'Clearance' or 'CoG' or 'Profile' or 'Lighting'
```

Comment:
I think we should use the CSVs we already have https://github.com/buildingSMART/ifc-gherkin-rules/blob/main/features/resources/valid_RepresentationIdentifier_IFC4.3.csv Not sure what that means for 2X3 for which we don't have a CSV. I guess leave it unconstrained for 2X3.